__tests__/cmds/docs/__snapshots__/index.test.ts.snap

```diff
@@ -35,7 +35,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Run \`docs\` command 🚀
-        uses: readmeio/rdme@7.8.9
+        uses: readmeio/rdme@v7
```

Comment:
How come all these snapshots are at v7 and not v8?

Reply:
I wanted to make sure the mocked value was different than the actual one:
https://github.com/readmeio/rdme/blob/e7d02ffbeda3c60a69042ac4a417a2c58fb7a968/__tests__/helpers/get-gha-setup.ts#L39